### PR TITLE
Add review for Portuguese translation of `softwarereview_reviewer.pt.Rmd`

### PR DESCRIPTION
**Pull request by pedropark99**
This PR is adding the first review of `softwarereview_reviewer.pt.Rmd`.

### CODE REVIEW

**Comment by arianacabral** [comment]
I can conduct the second review of this chapter.

**Comment by pedropark99** [comment]
Hey @arianacabral ! Thank you very much for your interest! If you want to, you can perform this second review by simply reviewing the changes in this PR 😉.

You review the changes by going into the "Files changed" panel in this PR. In other words, go to this link and review the changes: https://github.com/ropensci/dev_guide/pull/851/files

**Comment by arianacabral** [comment]
> Hey @arianacabral ! Thank you very much for your interest! If you want to, you can perform this second review by simply reviewing the changes in this PR 😉.
> 
> You review the changes by going into the "Files changed" panel in this PR. In other words, go to this link and review the changes: https://github.com/ropensci/dev_guide/pull/851/files

Hi, @pedropark99! 

Thank you for providing the details! 

I’ll review the changes in the "Files changed" panel and will share my feedback as soon as possible 😊

File: softwarereview_reviewer.pt.Rmd [hunk 1]
```diff
@@ -7,69 +7,68 @@ aliases: reviewerguide.html
...
 
-### Experiência de revisores anteriores {#experience-from-past-reviewers}
+### Experiências de revisores(as) anteriores {#experience-from-past-reviewers}
 
-Os avaliadores iniciantes podem achar útil ler (sobre) algumas avaliações anteriores. Em geral, você pode encontrar tópicos de envio de pacotes integrados [aqui](https://github.com/ropensci/software-review/issues?q=is%3Aissue+is%3Aclosed+label%3A6%2Fapproved). Aqui estão alguns exemplos escolhidos de avaliações (observe que suas avaliações não precisam ser tão longas quanto esses exemplos):
+Os(as) avaliadores(as) iniciantes podem achar útil ler (sobre) algumas avaliações anteriores. Em geral, você pode encontrar os tópicos de envio de pacotes que já foram integrados [aqui](https://github.com/ropensci/software-review/issues?q=is%3Aissue+is%3Aclosed+label%3A6%2Fapproved). Aqui estão alguns exemplos de avaliações (observe que suas avaliações não precisam ser tão longas quanto esses exemplos):
 
 - `rtika` [revisão 1](https://github.com/ropensci/software-review/issues/191#issuecomment-367166658) e [revisão2](https://github.com/ropensci/software-review/issues/191#issuecomment-368254623)
```
Comment (arianacabral) [inline softwarereview_reviewer.pt.Rmd:73]: Acredito que talvez faça mais sentido, para quem está lendo, manter essa estrutura. Se achar adequado, pode alterar as demais para que fiquem todos os exemplos no mesmo padrão.

```suggestion
- [revisão 1](https://github.com/ropensci/software-review/issues/191#issuecomment-367166658) e [revisão2](https://github.com/ropensci/software-review/issues/191#issuecomment-368254623) de `rtika`
```

**Review by arianacabral — CHANGES_REQUESTED** [review]
Hi, @pedropark99!

I have completed the second review.

I must say that I was very impressed with the first review; it was very good.

In this second review, I made just a few minor adjustments to improve some sentences and to standardize the reference to certain terms. However, please feel free to modify any of my suggestions if you think it’s necessary. 

For some points where I had doubts, I’ve left comments.

File: softwarereview_reviewer.pt.Rmd [hunk 2]
```diff
@@ -7,69 +7,68 @@ aliases: reviewerguide.html
 ```{block, type="summaryblock"}
 Obrigado por você ter aceitado revisar um pacote para a rOpenSci! Este capítulo consiste em nossas diretrizes para [preparar](#preparereview), [enviar](#submitreview) e [acompanhar](#followupreviewer) a sua revisão.
 
-Você pode entrar em contato com o editor responsável pela submissão para esclarecer qualquer dúvida que tenha sobre o processo ou sobre a sua revisão.
+Você pode entrar em contato com o(a) editor(a) responsável pela submissão para esclarecer qualquer dúvida que tenha sobre o processo ou sobre a sua revisão.
 
-Esforce-se para concluir sua avaliação em até 3 semanas após a aceitação da solicitação de avaliação. Procuraremos lembrar os revisores das datas de vencimento futuras e passadas. Os editores podem designar
-Os editores podem designar revisores adicionais ou alternativos se uma revisão estiver excessivamente atrasada.
+Tente ao máximo concluir sua avaliação em até 3 semanas após o aceite da solicitação de avaliação. Nós vamos lembrar os revisores das datas de vencimento, tanto futuras quanto passadas. Os(as) editores(as) podem designar revisores(as) adicionais ou alternativos se uma revisão estiver muito atrasada.
```
[the remark this answers is not in the document]
Comment (pedropark99) [reply softwarereview_reviewer.pt.Rmd:12]: I just noticed the lack of "(as)" in this other instance.

```suggestion
Tente ao máximo concluir sua avaliação em até 3 semanas após o aceite da solicitação de avaliação. Nós vamos lembrar os(as) revisores(as) das datas de vencimento, tanto futuras quanto passadas. Os(as) editores(as) podem designar revisores(as) adicionais ou alternativos se uma revisão estiver muito atrasada.
```

File: softwarereview_reviewer.pt.Rmd [hunk 3]
```diff
@@ -7,69 +7,68 @@ aliases: reviewerguide.html
...
-Obrigado pelo seu desejo de participar da revisão por pares do software rOpenSci como revisor!
+Obrigado pelo seu desejo de participar como revisor(a) da revisão por pares de software da rOpenSci!
 
-Por favor, preencha nosso [formulário de voluntariado](https://ropensci.org/software-reviewer).
+Por favor, preencha o nosso [formulário de voluntariado](https://ropensci.org/software-reviewer).
 
-Se você vir um envio atual que seja particularmente relevante para seus interesses, envie um e-mail para `info@ropensci.org` incluindo o nome do pacote, o URL do problema do envio e o nome do editor designado.
-No entanto, lembre-se de que os convites para revisores são mantidos a critério do editor, e é bem possível que o editor já tenha enviado um e-mail para as pessoas.
-Não se voluntarie para todas as edições e não se voluntarie por meio da interface do GitHub.
+Se você ver um envio atual que seja particularmente relevante para os seus interesses, envie um e-mail para `info@ropensci.org` incluindo o nome do pacote, a URL do *issue* que corresponde a este envio e o nome do(a) editor(a) responsável.
```
[the remark this answers is not in the document]
Comment (pedropark99) [reply softwarereview_reviewer.pt.Rmd:28]: O termo "issue" está sendo mantido intacto. Pois a tradução direta "problema" é muito esquisita, e não ajuda muito a entendermos o significado original (isto é, uma issue do GitHub).

File: softwarereview_reviewer.pt.Rmd [hunk 4]
```diff
@@ -7,69 +7,68 @@ aliases: reviewerguide.html
...
+As revisões devem ser baseadas na versão mais recente da *branch* `main` no GitHub, a menos que seja indicado de alguma outra forma pelos(as) autores(as) do pacote.
+Todos os envios criam um relatório detalhado sobre a estrutura e a funcionalidade do pacote enviado. Tal relatório é gerado pelo nosso pacote [`pkgcheck`](https://docs.ropensci.org/pkgcheck/). Se o pacote tiver sido alterado substancialmente desde as últimas verificações, você pode solicitar uma nova verificação com o comando `@ropensci-review-bot check package`. Observe que, ao instalar o pacote para revisá-lo, você deve se certificar de que todas as dependências estão disponíveis em sua máquina (por exemplo, você pode executar `pak::pak()` para isso).
```
[the remark this answers is not in the document]
Comment (pedropark99) [reply softwarereview_reviewer.pt.Rmd:37]: Boa! Excelente essa mudança

File: softwarereview_reviewer.pt.Rmd (hunk 1 above)
Comment (pedropark99) [reply softwarereview_reviewer.pt.Rmd]: Faz sentido mesmo mudar essa estrutura dessa forma.

**Comment by pedropark99** [comment]
First of all, thank you very much @arianacabral for your review!

@yabellini  the two reviews are done, we can proceed with this PR.

**Pull request merged**